core-bundle/src/Command/AbstractLockedCommand.php

```diff
@@ -70,7 +70,7 @@ final protected function execute(InputInterface $input, OutputInterface $output)
     /**
      * @return int
      */
-    abstract protected function executeLocked(InputInterface $input, OutputInterface $output);
+    abstract protected function executeLocked(InputInterface $input, OutputInterface $output)/*: int*/;
```

Comment:
This class will be removed in Contao 5.0 anyway btw.

Reply:
Reverted in 8a11b062f49367e50172992bca97f89b629824ad.